crates/katana/executor/src/implementation/sir/mod.rs

```diff
@@ -248,14 +250,14 @@
             Ok((info, fee)) => {
                 // if the transaction was reverted, return as error
                 if let Some(reason) = info.revert_error {
-                    info!(target: "executor", "fee estimation failed: {reason}");
+                    info!(target: LOG_TARGET, reason = %reason, "Fee estimation failed.");
                     Err(ExecutionError::TransactionReverted { revert_error: reason })
                 } else {
                     Ok(fee)
                 }
             }
             Err(e) => {
-                info!(target: "executor", "fee estimation failed: {e}");
+                error!(target: LOG_TARGET, error = %e, "Estimating fee.");
```

Comment:
similar to [comment](https://github.com/dojoengine/dojo/pull/1702#discussion_r1547225893)